brian2tools/mdexport/mdexporter.py

```diff
@@ -14,7 +14,7 @@ class MdExporter(BaseExporter):
     """
 
     def build(self, direct_call=True, debug=False, expander=None,
-              filename=None, format=None, template_type=None):
+              filename=None, Additionalformat=None, template_type=None):
```

Comment:
Please rename to `additional_formats` (see [PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names))